### DISCUSSION TITLE
Request for Cancellation and Refund of Enterprise Account

### DISCUSSION THREAD

**Discussion by Ian-SeoJaeMin** [community/community#135717] [Enterprise] [answered]
### Select Topic Area

General

### Body

I mistakenly created an Enterprise account on my personal GitHub account instead of using my company account. This error has led to an unintended charge on my personal account, which should have been processed under the company’s billing information.

Despite submitting three tickets, I have not yet received a response. I mistakenly created an Enterprise account on my personal GitHub account instead of using my company account, resulting in an unintended charge on my personal account.

https://support.github.com/ticket/personal/0/2936822

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ian-SeoJaeMin, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
